apps/privacy-panel/js/root/main.js

```diff
@@ -43,8 +44,16 @@ function(panels, SettingsListener, SettingsHelper) {
 
       // Reset launch flag when app is not active.
       window.addEventListener('blur', function() {
+        this.fromSettings = this.panel.dataset.settings;
         SettingsHelper('privacypanel.launched.by.settings').set(false);
-      });
+      }.bind(this));
```

Comment:
you can use arrow function and by doing so, you don't have to `bind(this)` anymore.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions